helm/trivy/values.yaml

```diff
@@ -37,3 +37,6 @@ trivy:
     privileged: false
     readOnlyRootFilesystem: true
     allowPrivilegeEscalation: false
+  
+  trivy:
+    dbRepository: gsoci.azurecr.io/giantswarm/trivy-db
```

Comment:
Same Aliyun question here

Reply:
Yes but here we don't have it split in registry and repository. Will check what we can do

Reply:
Unfortunately here there is now way for us to split this template into two, so we need to overwrite the value directly at the catalog level